lib/get-cwd.js

```diff
@@ -3,29 +3,13 @@
 const fs = require('fs');
 const path = require('path');
 
+const runInterpolation = require('./run-interpolation');
 const ValidationError = require('./validation-error');
 
 module.exports = function getCwd(dir, options) {
   return Promise.resolve()
     .then(() => {
-      const values = Object.assign({}, options.command, options, { dir });
-
-      let value = options.path;
-      let match;
-
-      do {
-        match = /{([a-zA-Z0-9_]*)}/.exec(value);
-
-        if (match) {
-          const key = match[1];
-          if (key in values) {
-            value = value.replace(match[0], values[key]);
-          } else {
-            throw new ValidationError(`terraform path template ${options.path} references missing value '${key}'`)
-          }
-        }
-      } while (match);
-
+      const value = runInterpolation('path', options, { dir });
```

Comment:
I would prefer, I think, just`

```js
interpolate(options.path, ...)
interpolate(options['init-opts'], ...)
```

so signature is (value, option, defaults) rather than property name as path